contents/python/introduction-to-python.md

```diff
@@ -48,4 +48,13 @@ To get started, take a look at:
 *   [matplotlib Tutorials](http://matplotlib.org/users/tutorials.html#introductory)
 *   [scikit-learn Basics](http://scikit-learn.org/stable/tutorial/basic/tutorial.html)
 *   [scikit-learn Machine Learning tutorials](http://scikit-learn.org/stable/tutorial/index.html) are also available for those looking to pick up machine learning.
-    *   If you learn best by working on projects, fret not! [Kaggle](http://kaggle.com) is a treasure trove of real-world datasets that you can hone your data science skills on.
\ No newline at end of file
+*   If you learn best by working on projects, fret not! [Kaggle](http://kaggle.com) is a treasure trove of real-world datasets that you can hone your data science skills on.
```

Comment:
I think this piece of instruction is meant for machine learning, hence it is in its rightful place, and shouldn't be amend as part of this commit.

Reply:
Hmm. I saw it as a general Data Science advice which matched the title Python for Data Science. But this might not be the intent of the previous author. Will amend it.